volttrontesting/platform/test_instance_setup.py

```diff
@@ -52,9 +52,10 @@
 def create_vcfg_vhome():
     debug_flag = os.environ.get('DEBUG', False)
     vhome = create_volttron_home()
-    yield vhome
+    vhome_dir = os.path.split(vhome)[0]
```

Comment:
ummm....gonna have to tell me about this one?

Reply:
Fixed.